src/features/cards/organisms/card-item.js

```diff
@@ -76,3 +86,48 @@ const CardBox = styled(Card)`
   max-height: 24rem;
   overflow-y: hidden;
 `
+
+const PopUpButton = () => {
+  const [popup, setPopup] = useState(false)
+
```

Comment:
I think we should create const here:

```js
const close = () => setOpened(opened => !opened)
```

And use `close` fn in `onClose` prop in `PopUp`